### DISCUSSION TITLE
Custom Presets for the Entity Hider

### DISCUSSION THREAD

**Discussion by SirZandro** [runelite/runelite#17611] [Ideas]
Would it be possible to create the ability for users to create their own presets for the entity hider to switch settings quickly rather than selecting individual boxes each time? Being able to flip between different presets would make it much more convenient to use.

**Comment by DominickCobb-rs** [comment]
The pluginhub plugin "Plugin presets" allows you to do this.

**Reply by SirZandro** [reply]
I was not aware of this pluggin! I just searched against Entity Hider when looking for related threads and it didn't dawn on me that this would be an option. This should work perfectly, thank you.
